addons/web_editor/tools.py

```diff
@@ -97,6 +101,16 @@ def get_video_url_data(video_url, autoplay=False, loop=False, hide_controls=Fals
             params['controls'] = 0
         if loop:
             params['loop'] = 1
+        groups = platform_match.groupdict()
+        if 'hash' in groups:
+            params['h'] = groups['hash']
+        elif 'params' in groups:
+            try:
+                url_params = parse_qs(groups['params'])
+                if 'h' in url_params:
+                    params['h'] = url_params['h'][0]
+            except ValueError:
+                pass
```

Comment:
I am still wondering if we shouldn't raise an explicit for the user to see why it didn't work. As you essentially give no feedback as to why the video isn't shown. Plus while checking the doc for `parse_qs` I saw that you didn't add the parameters that are able to throw such an error, so is there really a possibility of throwing such an error ?

Reply:
As the hash parameter (`h`) is optional and we can not really tell if the video is unlisted or not, I would not throw any error. I will just try to add the parameter if it is provided and that's it. I will use the parameter `strict_parsing` on `parse_qs` to avoid the useless try catch.

Reply:
But you do not need to set it to False as it is the [default value](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.parse_qs), only removing the try except would have been sufficient